pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -14,131 +14,87 @@ package devworkspace
-	DevWorkspaceViewWorkspacesClusterRoleFile = DevWorkspaceTemplates + "/devworkspace-controller-view-workspaces.ClusterRole.yaml"
-	DevWorkspaceEditWorkspacesClusterRoleFile = DevWorkspaceTemplates + "/devworkspace-controller-edit-workspaces.ClusterRole.yaml"
-	DevWorkspaceRoleBindingFile               = DevWorkspaceTemplates + "/devworkspace-controller-leader-election-rolebinding.RoleBinding.yaml"
-	DevWorkspaceClusterRoleBindingFile        = DevWorkspaceTemplates + "/devworkspace-controller-rolebinding.ClusterRoleBinding.yaml"
-	DevWorkspaceProxyClusterRoleBindingFile   = DevWorkspaceTemplates + "/devworkspace-controller-proxy-rolebinding.ClusterRoleBinding.yaml"
-	DevWorkspaceWorkspaceRoutingCRDFile       = DevWorkspaceTemplates + "/devworkspaceroutings.controller.devfile.io.CustomResourceDefinition.yaml"
-	DevWorkspaceTemplatesCRDFile              = DevWorkspaceTemplates + "/devworkspacetemplates.workspace.devfile.io.CustomResourceDefinition.yaml"
-	DevWorkspaceCRDFile                       = DevWorkspaceTemplates + "/devworkspaces.workspace.devfile.io.CustomResourceDefinition.yaml"
-	DevWorkspaceConfigMapFile                 = DevWorkspaceTemplates + "/devworkspace-controller-configmap.ConfigMap.yaml"
-	DevWorkspaceServiceFile                   = DevWorkspaceTemplates + "/devworkspace-controller-manager-service.Service.yaml"
-	DevWorkspaceDeploymentFile                = DevWorkspaceTemplates + "/devworkspace-controller-manager.Deployment.yaml"
-	DevWorkspaceIssuerFile                    = DevWorkspaceTemplates + "/devworkspace-controller-selfsigned-issuer.Issuer.yaml"
-	DevWorkspaceCertificateFile               = DevWorkspaceTemplates + "/devworkspace-controller-serving-cert.Certificate.yaml"
-
 	WebTerminalOperatorSubscriptionName = "web-terminal"
 	WebTerminalOperatorNamespace        = "openshift-operators"
 )
 
-type Object2Sync struct {
-	obj     metav1.Object
-	hash256 string
-}
-
 var (
-	// cachedObjects
-	cachedObj = make(map[string]*Object2Sync)
-	syncItems = []func(*deploy.DeployContext) (bool, error){
-		createDwNamespace,
-		syncDwService,
-		syncDwServiceAccount,
-		syncDwClusterRole,
-		syncDwProxyClusterRole,
-		syncDwEditWorkspacesClusterRole,
-		syncDwViewWorkspacesClusterRole,
-		syncDwRole,
-		syncDwRoleBinding,
-		syncDwClusterRoleBinding,
-		syncDwProxyClusterRoleBinding,
-		syncDwIssuer,
-		syncDwCertificate,
-		syncDwCRD,
-		syncDwTemplatesCRD,
-		syncDwWorkspaceRoutingCRD,
-		syncDwConfigMap,
-		syncDwDeployment,
-	}
-
 	// Exits the operator after successful fresh installation of the devworkspace.
 	// Can be replaced with something less drastic (especially useful in tests)
-	afterInstall = func() {
-		logrus.Warn("Exitting the operator after DevWorkspace installation. DevWorkspace support will be initialized on the next start.")
+	restartCheOperator = func() {
+		logrus.Warn("Exiting the operator after DevWorkspace installation. DevWorkspace support will be initialized on the next start.")
 		os.Exit(1)
```

Comment:
Outside of this PR, I know, but should this point really be `exit(1)`? It's an expected condition, not an error.

Reply:
Agree. Exit(0) sounds better

Reply:
yeah, good catch and I'll address this since I have a few more minor stuff to handle, so I can't avoid returning PR checks